health/egov-persister/census-service-persister.yml

```diff
@@ -0,0 +1,98 @@
+      isAuditEnabled: true
+      module: CENSUS
+      objectIdJsonPath: $.id
+      tenantIdJsonPath: $.tenantId
+      userUuidJsonPath: $.auditDetails.createdBy
+      transactionCodeJsonPath: $.id
+      auditAttributeBasePath: $.Census
+      queryMaps:
+        - query: INSERT INTO health.census(id, tenant_id, hierarchy_type, boundary_code, type, total_population, effective_from, effective_to, source, status, assignee, materialized_path, additional_details, created_by, created_time, last_modified_by, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+          basePath: $.Census
+          jsonMaps:
+            - jsonPath: $.Census.id
+            - jsonPath: $.Census.tenantId
+            - jsonPath: $.Census.hierarchyType
+            - jsonPath: $.Census.boundaryCode
+            - jsonPath: $.Census.type
+            - jsonPath: $.Census.totalPopulation
+            - jsonPath: $.Census.effectiveFrom
+            - jsonPath: $.Census.effectiveTo
+            - jsonPath: $.Census.source
+            - jsonPath: $.Census.status
+            - jsonPath: $.Census.assignee
+            - jsonPath: $.Census.boundaryCodeAncestralMaterializedPath
+            - jsonPath: $.Census.additionalDetails
+              type: JSON
+              dbType: JSONB
+            - jsonPath: $.Census.auditDetails.createdBy
+            - jsonPath: $.Census.auditDetails.createdTime
+            - jsonPath: $.Census.auditDetails.lastModifiedBy
+            - jsonPath: $.Census.auditDetails.lastModifiedTime
+
+        - query: INSERT INTO health.population_by_demographics(id, census_id, demographic_variable, population_distribution, created_by, created_time, last_modified_by, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?);
+          basePath: $.Census.populationByDemographics.*
+          jsonMaps:
+            - jsonPath: $.Census.populationByDemographics.*.id
+            - jsonPath: $.Census.id
+            - jsonPath: $.Census.populationByDemographics.*.demographicVariable
+            - jsonPath: $.Census.populationByDemographics.*.populationDistribution
+              type: JSON
+              dbType: JSONB
+            - jsonPath: $.Census.auditDetails.createdBy
+            - jsonPath: $.Census.auditDetails.createdTime
+            - jsonPath: $.Census.auditDetails.lastModifiedBy
+            - jsonPath: $.Census.auditDetails.lastModifiedTime
+
+    - version: 1.0
+      name: census
+      description: Updates census details
+      fromTopic: census-update-topic
+      isTransaction: true
+      isAuditEnabled: true
+      module: CENSUS
+      objectIdJsonPath: $.id
+      tenantIdJsonPath: $.tenantId
+      userUuidJsonPath: $.auditDetails.createdBy
+      transactionCodeJsonPath: $.id
+      auditAttributeBasePath: $.Census
+      queryMaps:
+        - query: UPDATE health.census SET total_population = ?, status = ?, assignee = ?, additional_details = ?, last_modified_by = ?, last_modified_time = ? WHERE id = ?;
```

Comment:
why is total population updateable? it should not be updateable

Reply:
> why is total population updateable? it should not be updateable

okay